src/axelrod_dojo/archetypes/fsm.py

```diff
@@ -83,10 +81,10 @@ def mutate_rows(rows, mutation_rate):
         return rows
 
     def mutate(self):
-        self.rows = self.mutate_rows(self.rows, self.mutation_rate)
-        if random.random() < self.mutation_rate / 10:
+        self.rows = self.mutate_rows(self.rows, self.mutation_probability)
```

Comment:
The scaling here was intentional so that some parts are mutated more often than others.